apps/user-profile/src/assets/i18n/en.json

```diff
@@ -223,16 +223,6 @@
       "CLOSE": "Close",
       "COPY": "Copy"
     },
-    "REQUEST_DATA_QUOTA_CHANGE": {
```

Comment:
You forgot to delete also duplication of `DATA_QUOTAS` and `OPT_OUT_MAILING_LISTS` in this file.